lib/screens/wallet/components/tokens_cards/tokens_cards.dart

```diff
@@ -83,13 +83,17 @@ class _TokenCardsState extends State<TokenCards> with AutomaticKeepAliveClientMi
                           title: 'Receive',
                           onPressed: () async {
                             final address = "5HGZfBpqUUqGY7uRCYA6aRwnRHJVhrikn8to31GcfNcifkym";
```

Comment:
test for the new validate function - we need to use this on all text fields where users enter account addresses